src/test/java/org/symphonyoss/symphony/messageml/elements/ElementTest.java

```diff
@@ -1729,7 +1730,10 @@ public void testTable() throws Exception {
     cell = row.getChildren().get(0);
     assertEquals("Element class", TableCell.class, cell.getClass());
     assertEquals("Element tag name", "td", cell.getMessageMLTag());
-    assertEquals("Element attributes", Collections.emptyMap(), cell.getAttributes());
+    final HashMap<String, String> expectedAttributeMap = new HashMap<>();
+    expectedAttributeMap.put("colspan", "6");
+    expectedAttributeMap.put("rowspan", "7");
+    assertEquals("Element attributes", expectedAttributeMap, cell.getAttributes());
```

Comment:
Also assert that the plaintext representation (context.getText()) doesn't contain those attrs by any chance.

Reply:
That is already asserted below this code